static/src/javascripts-legacy/projects/common/modules/atoms/story-questions.js

```diff
@@ -35,6 +39,29 @@ define([
                     bean.on(el, 'click', askQuestion);
```

Comment:
No worries, it's a minor detail in this situation. When you have dozens of questions for instance, you want a single listener on their common ancestor and then figure out the question that was clicked by doing a simple array lookup. I can show you later.